### ISSUE TITLE
Show mark after the end of every block (1KB, 4KB...)

### ISSUE THREAD

**Issue by juliangruber** [juliangruber/hex-dump#25]
@trufae which blocks should be delimited exactly?

**Comment by trufae** [comment]
Usually 4KB and 16KB but i think we can copy this from the old vialab hex
editor options

**Comment by juliangruber** [comment]
does anyone have a screenshot of this?

**Comment by trufae** [comment]
i do
<img width="1021" alt="screen shot 2016-02-14 at 22 34 40" src="https://cloud.githubusercontent.com/assets/6431515/13036575/36c569d8-d36b-11e5-905b-9f4c1bdd6d74.png">

**Comment by trufae** [comment]
Oops. when i did the screenshot, the dropdown menu got shy

**Comment by trufae** [comment]
Fixed with the radical hacking technique of changing the focus
<img width="400" alt="screen shot 2016-02-14 at 22 37 50" src="https://cloud.githubusercontent.com/assets/6431515/13036587/9ddced9e-d36b-11e5-8b9c-8c92ff9cf643.png">

**Comment by juliangruber** [comment]
thank you. can you also show me how the rendered hex dump looks different with any of those options active?

**Comment by trufae** [comment]
Nothing changes, which is imho confusing. imho it will be good to show a
<hr size=1 /> after every block or so

**Comment by juliangruber** [comment]
ah, gotcha! thanks, i understand the problem now

**Comment by tomanderson2** [comment]
@trufae afaik the `display bytes` option in the hex viewer just alters how much data is loaded into the UI, nothing fancy :) I like the idea of marking blocks though +1

**Comment by radare** [comment]
yeah it looks more like a bad design decision to include that option to just change what's loaded. having block separators and doing dynamic loading is way better option imho

**Comment by tomanderson2** [comment]
Yes, thats something that should NOT be copied into this new hex viewer. I think this was something required a while back for users with slow machines due to how data was loaded. Dynamic loading FTW
